src/main/webapp/i18n/de/exam.json

```diff
@@ -239,6 +239,8 @@
             }
         },
         "examManagement": {
+            "reviewWarning": "Ende der Klausureneinsicht fehlt",
+            "reviewExpl": "Ist ein Beginn der Klausureinsicht definiert, muss auch ein Ende definiert werden.",
```

Comment:
I think we are not using abbreviations here, is there a certain reason why you use Expl instead of Explanation? Anyhow, I would call it reviewDatesInvalidWarning or something similar, as it more precisely describes where the text is being used.

Reply:
You are right, this would make it more clear. I changed them into reviewDatesInvalidWarning and reviewDatesInvalidExplanation. I hope you agree with that renaming.